### PR TITLE
Add more simple views

### PR DESCRIPTION
**Pull request by Jeadie**
- 14 views from [pkg/dremio/datasets/query.yaml](https://github.com/spicehq/data-platform/blob/5ca557386c9189d7fc94395bf35f823055ee41df/pkg/dremio/datasets/query.yaml) that are:
     - Simple views
     - Only depend on other "public" views. (i.e. no `backfillpostgres`, etc).  
     - No reflections / accelerations

### CODE REVIEW

File: .spice/datasets/eth_nft_airdrop_transfers.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+name: spiceai.datasets.eth_nft_airdrop_transfers
+type: view
+migrations:
+  - name: create_view
+    sql: |
+      SELECT nt.token_address, nt.token_id, t.from_address AS tx_from_address, t."value" AS tx_value, nt.from_address, nt.to_address, nt.transaction_hash, nt.block_number, nt.block_timestamp, nt.block_hash
```
Comment (lukekim) [inline .spice/datasets/eth_nft_airdrop_transfers.yaml:6]: typo

Comment (Jeadie) [reply .spice/datasets/eth_nft_airdrop_transfers.yaml:6]: Which/where is the typo? This is from [`pkg/dremio/datasets/query.yaml`](https://github.com/spicehq/data-platform/blob/d04fc996f5c4f9506afc3816fd2d071d81d763ae/pkg/dremio/datasets/query.yaml#L77)

File: .spice/datasets/eth_nft_owners.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+type: view
+migrations:
+  - name: create_view
+    sql: |
+      WITH ranked_trades AS (
+          SELECT token_address, token_id, to_address, block_timestamp, transaction_hash, ROW_NUMBER() OVER (PARTITION BY token_address, token_id ORDER BY block_number DESC) AS trade_rank
+          FROM eth.nft_transfers
+      )
+      SELECT token_address, token_id, to_address as "owner", block_timestamp, transaction_hash
+      FROM ranked_trades WHERE trade_rank =
```
Comment (phillipleblanc) [inline .spice/datasets/eth_nft_owners.yaml]: This is missing a comparison?

File: .spice/datasets/eth_nft_transfers.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+name: spiceai.datasets.eth_nft_transfers
+type: view
+migrations:
+  - name: create_view
+    sql: |
+      WITH token_transfers_nft AS
+        (SELECT block_number, block_timestamp, block_hash, token_address, from_address, to_address, transaction_hash, token_id
+        FROM eth.token_transfers
+        WHERE token_standard = 'erc721' OR token_standard = 'erc1155')
+
```
Comment (phillipleblanc) [inline .spice/datasets/eth_nft_transfers.yaml]:
```suggestion
      WITH token_transfers_nft AS (
        SELECT block_number, block_timestamp, block_hash, token_address, from_address, to_address, transaction_hash, token_id
        FROM eth.token_transfers
        WHERE token_standard = 'erc721' OR token_standard = 'erc1155'
      )

```

**Review by phillipleblanc — APPROVED** [review]

**Pull request merged**